charts/flyte-core/templates/clusterresourcesync/deployment.yaml

```diff
@@ -55,9 +55,9 @@ spec:
             name: flyte-clusterresourcesync-config
           name: config-volume
         {{- if gt (len .Values.configmap.clusters.labelClusterMap) 0 }}
-        - name: flyte-admin-secrets
+        - name: cluster-secrets
           secret:
-            secretName: flyte-admin-secrets
+            secretName: cluster-secrets
```

Comment:
where is this secret created?  is the name of the secret "cluster-secrets"?  would you mind linking me to where that's set?

Reply:
Sure, it comes here from the guide
https://docs.flyte.org/en/latest/deployment/deployment/multicluster.html#deployment-deployment-multicluster

Reply:
Ideally that should not be created manually but as part of the helm chart and only filled in with values by users, but for now this fixes the current setup

Reply:
i'm still confused by this.  i don't see the text "cluster-secrets" on that page.  I only see the `cluster-credentials` which should be covered by the additional values.

Reply:
I think `cluster-credentials` is what you mean here @gdabisias right?

Reply:
Besides naming, I think this change makes sense. It enables `syncresources` to mount a pre-created Secret containing only the credentials for the multiple clusters the control plane will connect to, instead of having to manually edit the `flyte-admin-secrets` which mounts additional sensitive data that doesn't seem to be needed by `syncresources`

Reply:
yeah this should be cluster-credentials; let me update that.

I think this should be fine as a first fix, but then we should have secrets that get created independently of the user code and they just get filled in by the user, like we do for the database secrets

Reply:
Fixed